pnpm-lock.yaml

```diff
@@ -4969,7 +4963,7 @@ packages:
       pretty-ms: 8.0.0
       strip-json-comments: 5.0.1
       summary: 2.1.0
-      typescript: 5.3.3
+      typescript: 5.2.2
```

Comment:
Hm, what happened here? Why is this not 5.3.3?

Reply:
Super odd:

```
/knip@3.8.1(@types/node@16.18.61)(typescript@5.2.2):
```

Probably we should upgrade TS in everything anyway.

Reply:
I ran `pnpm i` and the top-level package.json has 5.2.2. I can change it to 5.3.3 I guess.

Reply:
No worries, I don't think it's a problem. My other PR updates knip anyway.